society-for-american-archaeology.csl

```diff
@@ -103,20 +91,31 @@
       <substitute>
         <names variable="editor"/>
         <names variable="translator"/>
+        <names variable="director"/>
       </substitute>
     </names>
-    <text macro="anon"/>
-    <text macro="recipient"/>
+    <text macro="title"/>
+    <choose>
+      <if type="map" match="any">
+        <text variable="publisher" prefix=" (" suffix=")"/>
+      </if>
+    </choose>
   </macro>
   <macro name="contributors-short">
     <names variable="author">
       <name form="short" and="text" delimiter=", "/>
+      <label form="short" prefix=", "/>
       <substitute>
         <names variable="editor"/>
         <names variable="translator"/>
       </substitute>
     </names>
-    <text macro="anon"/>
+    <text macro="title"/>
```

Comment:
Is this right? This would make the title appear for every in-text citation -- I assume you want this in substitute?